common/djangoapps/student/forms.py

```diff
@@ -144,6 +148,13 @@ class AccountCreationForm(forms.Form):
             "min_length": _PASSWORD_INVALID_MSG,
         }
     )
+    password_copy = forms.CharField(
```

Comment:
@cahrens as far as I know there are 2 sets of views for the registration form - old and new (with microsites old views are used), so I'll need to implement "Retype password" in 2 places. The current implementation of "Retype password" works with and without microsites. Also all validation messages have the same format, all validation code is in the same place. Will you insist on re-implementing "Retype password" on the front-end? I'm postponing fixing of broken jenkins tests.